source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,197 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can configure Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having to 
+manage your own Kafka cluster. 
+
+Before you Begin
+----------------
+
+- You must have Relational Migrator installed on your machine. To download 
+  Relational Migrator, see the `Download Center 
+  <https://www.mongodb.com/try/download/relational-migrator>`__.
+
+- Due to restrictions by Confluent, you are only able to use Basic, Standard, and 
+  Dedicated clusters. For most use cases, the basic and standard cluster are enough 
+  to handle migrating large volumes of data.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal 
+
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
+   .. step:: Enable schema registry
+
+      Enable schema registry and select a region.
```

Comment:
[question] As the user, how do I do this?

Reply:
This window appears after you create your environment, so as a user you do not have to click through anything. After you finish the first step it automatically prompts you to do this.